frontend/src/component/Signup.jsx

```
@@ -0,0 +1,200 @@
+              ) : (
+                <EyeIcon size={20} />
+              )}
+            </button>
+          </div>
+          <button
+            onClick={handleSubmit}
+            className="w-full py-2 bg-blue-600 hover:bg-blue-700 rounded-md text-white font-semibold transition flex justify-center items-center"
+            disabled={loading}
+          >
+            {loading ? (
+              <span className="animate-spin h-5 w-5 border-2 rounded-full border-white"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)